src/FixedDataTableBufferedRows.js

```diff
@@ -78,36 +79,30 @@ class FixedDataTableBufferedRows extends React.Component {
     var rowClassNameGetter = props.rowClassNameGetter || emptyFunction;
     var rowsToRender = this.props.rowsToRender || [];
 
-    this._staticRowArray.length = rowsToRender.length;
-    var baseOffsetTop = props.offsetTop - props.scrollTop;
+    if (!props.isScrolling) {
```

Comment:
This is where handling of old rows are done. We just clone the element and set the `visible` prop to `false`